server/jira.js

```diff
@@ -268,6 +270,39 @@ Jira.verifyTicketOnDev = function(ticket) {
   Jira.makePostRequest(url, data);
 };
 
+Jira.reOpenAndCommentTicket = function(ticket, failReason) {
+  if (ticket.statusName === Config.jira.toDoStatusName) {
+    return;
+  }
+
+  if (ticket.isRegression) {
+    return;
+  }
+
+  var jiraId = ticket.jiraId
+  var url =  "issue/" + jiraId + "/transitions"
+  var data = {
+    transition: {
+      id: 201
```

Comment:
can you add a comment as to what this is transitioning to? For extra credit can you add a comment to the the other transition call that is happening when a ticket is verified?